### ISSUE TITLE
Does committee gem have plans to support OpenAPI 3.0?

### ISSUE THREAD

**Issue by akichim21** [interagent/committee#126]
Does committee gem have plans to support OpenAPI 3.0?

I would be happy if you would consider it.

**Comment by brandur** [comment]
Hi @akichim21, Committee should absolutely support OpenAPI 3.0, but so far it's just been a problem of time — I don't use Committee for of my own active projects at home or work right now, so I've never had the chance to put it in.

If you'd be willing to take a look at implementing it, I don't think it would be all that difficult (there's already a driver system in place so that formats can be changed in and out), and I'd will be willing to help out in shepherding it along.

**Comment by akichim21** [comment]
Hi @brandur, thank you for your reply.
I also want to contribute when I have time.

**Comment by yhirano55** [comment]
@brandur Hi, I'm eagerly using this library, and I appreciate it so much for your contributions. But if you don't have enough time to spare to maintain this gem, it's better to add maintainers, I think. What do you think about it?

**Comment by brandur** [comment]
> @brandur Hi, I'm eagerly using this library, and I appreciate it so much for your contributions. But if you don't have enough time to spare to maintain this gem, it's better to add maintainers, I think. What do you think about it?

Hi @yhirano55, yes I'm doing a very poor job on the maintenance front. I am interested in finding collaborators/maintainers. Are you volunteering? ;) cc @KentaYoshitani @ota42y @tzmfreedom as some recent contributors as well.

The first step will be to start getting some eyes on @KentaYoshitani's addition in #128 (which I am also going to try to do very soon).

**Comment by yhirano55** [comment]
@brandur Thanks for your comment. Nice idea 👏

**Comment by KentaYoshitani** [comment]
Thank you for replying @brandur !
If you are okay with me,  I'll help you as a maintainer!

**Comment by brandur** [comment]
@yhirano55 @KentaYoshitani Excellent. Okay, I've created a new team for @interagent/committee-maintainers and invited you both to it. It has read/write access, but let's still try to send pull requests to each other for any changes (at least while we're ramping up).

**Comment by ota42y** [comment]
@brandur 
Hi! I'm interested in maintenance of this gem. 👍   
So It's nice idea, I'll help you too. 🙋

**Comment by brandur** [comment]
@ota42y Great! I just invited you as well.

**Comment by ota42y** [comment]
I received invitation 👍

**Comment by ahx** [comment]
Hey folks, I would like to help out with OAS3 support, but I am not sure on which branch I should start. – Since some tests are red (which is good, I think) on `feature/working_openapi3`, I assume that there is still work to do. Do you have a checklist of things that are missing to make OAS3 work?
One thin I noticed is:
- [ ] Make assert_schema_conform support OAS3

Anything else (except the failing tests on `feature/working_openapi3`?

**Comment by ota42y** [comment]
@ahx It's good news!

Now I create new OAS3 parser and replaced this PR (https://github.com/interagent/committee/pull/163)

And I tried using this version committee in my application on staging env.
So I completed main tasks for  OAS3 support
But I have many todo tasks :(
(Because my application don't use these options so I postponed)

So we need these steps to release OAS3 support
- [ ] fix failed test
  - when https://github.com/interagent/committee/pull/163 was merged and https://github.com/interagent/committee/pull/145 passed test, I wan't to merge OAS3 feature to 3-0 branch
- [ ] implement todo tasks
  - I think that it is good to release even if it does not finish all.
- [ ] merge master branch to 3-0 branch
  - few change added in master so we add these to 3-0 branch
